src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/custom-anomalies.mdx

```diff
@@ -0,0 +1,104 @@
+* For NRQL alerts, see the [allowed types of NRQL queries](/docs/alerts/new-relic-alerts/defining-conditions/create-alert-conditions-nrql-queries#threshold-types).
+* If the alert condition applies to multiple apps, you can select a choice from the dropdown above the chart to use different metrics. (Not applicable for NRQL alert conditions.)
+
+### Faceted anomaly conditions [#anomaly-facets]
+
+Once you've run a faceted NRQL query for a anomaly condition, you can scope your results to a single anomaly.
+
+<img
+  title="Faceted anomalies"
+  alt="A screenshot depicting faceted anomalies"
+  src={alertsFacetedAnomalies}
+/>
+
+The single time series shows the anomaly, threshold band, and one or more violation regions, if there are any.
+
+<img
+  title="Configured faceted anomalies"
+  alt="A screenshot depicting a deeper dive into facted anomalies "
+  src={alertsFacetedAnomaliesTwo}
+/>
+
+### Anomaly direction: select upper or lower ranges
+
+You can choose whether you want the condition to violate for behavior that goes above the predicted value ("upper") or that goes below the predicted value ("lower"), or that goes either above or below. You choose these with the  prediction direction selector.
+
+Example use cases for this:
+* You might use the Upper setting for a data source like error rate, because you generally are only concerned if it goes up, and aren't concerned if it goes down.
+* You might use the Lower setting for a data source like throughput, because sudden upward fluctuations are quite common, but a large sudden downswing would be a sign of a problem.
+
+Here are examples of how large fluctuations in your data would be treated under the different anomaly direction settings. The red areas represent violations
+
+<img
+  title="A screenshot of anomaly details in the New Relic UI"
+  alt="A screenshot demonstrating how to select upper and lower ranges for anomalies"
+  src={alertsAnomaliesSetUpperandLowerRanges}
+/>
+
+### Rules governing calculation of the predicted value
+
+The algorithm for calculating the prediction is mathematically complex. Here are some of the major rules governing its predictive abilities:
+
+* **Age of data** On initial creation, the  prediction is calculated using between 1 to 4 weeks of data, depending on data availability and  prediction type. After its creation, the algorithm will take into account ongoing data fluctuations over a long time period, although greater weight is given to more recent data. For data that has only existed for a short time, the  predicted value will likely fluctuate a good deal and not be very accurate. This is because there is not yet enough data to determine its usual values and behavior. The more history the data has, the more accurate the  prediction and thresholds will become.
```

Comment:
double spaces ahead of each use of prediction and predicted. There are 4 in this section.

Reply:
"The algorithm will take into account" should be "the algorithm takes into account" maybe?

Reply:
"there is not" to "there isn't"